landing.html

```diff
@@ -1,7 +1,7 @@
-<!DOCTYPE html>
-<html>
-    <body>
-        <link rel="stylesheet" href="stylesheet.css">  

```

Comment:
this change is doodoo and you should feel bad

Reply:
cool bro, so are you